libcst/codemod/commands/rename.py

```diff
@@ -0,0 +1,345 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+import argparse
+from typing import Callable, Optional, Sequence, Set, Tuple, Union
+
+import libcst as cst
+from libcst.codemod import CodemodContext, VisitorBasedCodemodCommand
+from libcst.codemod.visitors import AddImportsVisitor, RemoveImportsVisitor
+from libcst.helpers import get_full_name_for_node
+from libcst.metadata import QualifiedNameProvider
+
+
+def leave_import_decorator(
```

Comment:
What's the value of making this a decorator as opposed to doing this inline or making a helper function?  Not opposed to it, but just want to understand =)

Reply:
There are so many codepaths that lead to a return statement in both `leave_Import` and `leave_ImportFrom` that I thought it would be easier to intercept it with a decorator :)